Update all dependencies

Abban opened the pull request:
Note: The typescript eslint plugin isn't ready for
eslint v9 yet, so we need to keep it at v8 for now

See: https://github.com/typescript-eslint/typescript-eslint/pull/9119

moiikana commented on desktop/Banner.jsx:
```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import ProgressBar from '../shared/components/ui/ProgressBar';
 import Footer from '../shared/components/ui/EasySelectFooter';
-import Infobox from '../shared/components/ui/Infobox';
+import Infobox from './components/ui/Infobox';
```
```suggestion
import Infobox from './components/ui/InfoBox';
```

Abban committed:
Update all dependencies

Note: The typescript eslint plugin isn't ready for
eslint v9 yet, so we need to keep it at v8 for now

See: https://github.com/typescript-eslint/typescript-eslint/pull/9119
